### PR TITLE
moar 1.8.1 (new formula)

### PR DESCRIPTION
**Pull request by ritzmann**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/moar.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Moar < Formula
+  desc "Nice to use pager for humans"
+  homepage "https://github.com/walles/moar"
+  url "https://github.com/walles/moar.git",
+    using:    :git,
+    tag:      "v1.8.1",
+    revision: "fec9ad814970ee1a271b19326214647460b3200c"
```
Comment (nandahkrishna) [inline Formula/moar.rb:7]: Can we not use the source archive instead?

File: Formula/moar.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  license "BSD-2-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    system "./build.sh"
+    bin.install "moar"
```
Comment (nandahkrishna) [inline Formula/moar.rb:14]: Let's build this the same way we do for other `go`-based formulae, take a look at `driftctl` for an example.

**Review by nandahkrishna — COMMENTED** [review]
Thanks for your PR, @ritzmann. I have a few suggestions. Once you've made these changes, please squash them with your existing commit, and use your existing commit message.

**Comment by ritzmann** [comment]
Thanks @nandahkrishna. The latest update should address all your comments.

File: Formula/moar.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.versionString=v#{version}
+    ].join(" ")
+
+    system "go", "build", *std_go_args(ldflags: ldflags)
```
Comment (nandahkrishna) [inline Formula/moar.rb:16]:
```suggestion
    ldflags = "-s -w -X main.versionString=v#{version}"
    system "go", "build", *std_go_args(ldflags: ldflags)
```

File: Formula/moar.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    ldflags = "-s -w -X main.versionString=v#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    # Test piping something through moar
+    (testpath/"test.file").write <<~EOS
+      tyre kicking
+    EOS
+    assert_equal "tyre kicking", shell_output("moar test.file").strip
```
Comment (nandahkrishna) [inline Formula/moar.rb:20]:
```suggestion
    (testpath/"test.txt").write <<~EOS
      tyre kicking
    EOS
    assert_equal "tyre kicking", shell_output("#{bin}/moar test.txt")
```
Sorry, forgot to add this suggestion last time. Should be good to merge once this is addressed.

**Review by nandahkrishna — APPROVED** [review]
Thanks for contributing, and for following the contributing guidelines too @ritzmann!

**Pull request closed without merging**